script/setup

```diff
@@ -14,6 +14,9 @@ fi
 
 script/bootstrap
 
+# We make sure the containers are mounted before any further action
+sleep 5
```

Comment:
Small nitpick, it might be nice to log something before sleeping here, like `echo "Waiting for containers to mount..."` or something like that.

Reply:
I agree !